### DISCUSSION TITLE
Add intel items to containers

### DISCUSSION THREAD

**Discussion by umichhaidn** [acemod/ACE3#8942] [Q&A]
I've been trying to add intel items to containers but noticed that the acex_intelitems_fnc_addIntel function prohibits adding intel to anything except units. Is there a reason for this? It would be nice to be able to hide intel in desks or safes.

**Comment by rautamiekka** [comment]
`ace_vehiclelock_fnc_addKeyForVehicle` suffers from the same problem.

**Comment by umichhaidn** [comment]
Okay so I figured out why this is. It's because the various ace intelItem functions rely on the magazine index (as returned by the magazineDetail function) of each intel item. And as far as I can tell, there is no way to actually check the mag index of magazines inside a container. Which is weird cause the mag id does seem to say with a magazine when you put it into a container and take it out again.
